code/game/objects/structures/detectiveboard.dm

```diff
@@ -0,0 +1,254 @@
+#define MAX_ICON_NOTICES 5
+
+/obj/structure/detectiveboard
+	name = "detective notice board"
+	desc = "A board for linking evidence to crimes."
+	icon = 'icons/obj/wallmounts.dmi'
+	icon_state = "noticeboard"
+	density = FALSE
+	anchored = TRUE
+	max_integrity = 150
+	req_access = ACCESS_DETECTIVE
+
+	var/attaching_evidence = FALSE
+	var/list/case_colors = list("red", "orange", "yellow", "green", "blue", "violet")
+	var/list/datum/case/cases = list()
+	var/current_case = 1
+	var/data_test
```

Comment:
This isn't used anywhere

Reply:
case_colors is used when choosing a color for the case folder

Reply:
I meant ``data_test``

Reply:
Deleted